src/kOS/Execution/ProgramContext.cs

```diff
@@ -111,13 +111,21 @@ public List<string> GetCodeFragment(int start, int stop)
         {
             var codeFragment = new List<string>();
 
-            const string FORMAT_STR = "{0,-20} {1,4}:{2,-3} {3:0000} {4} {5}";
-            codeFragment.Add(string.Format(FORMAT_STR, "File", "Line", "Col", "IP  ", "opcode", "operand"));
-            codeFragment.Add(string.Format(FORMAT_STR, "----", "----", "---", "----", "---------------------", ""));
+            const string FORMAT_STR = "{0,-20} {1,4}:{2,-3} {3:0000} {4} {5} {6}";
+            codeFragment.Add(string.Format(FORMAT_STR, "File", "Line", "Col", "IP  ", "label", "opcode", "operand"));
+            codeFragment.Add(string.Format(FORMAT_STR, "----", "----", "---", "----", "-------------------------------", "", ""));
 
             for (int index = start; index <= stop; index++) {
                 if (index >= 0 && index < Program.Count) {
-                    codeFragment.Add(string.Format(FORMAT_STR, Program[index].SourceName, Program[index].SourceLine, Program[index].SourceColumn, index, Program[index], (index == InstructionPointer ? "<<--INSTRUCTION POINTER--" : "")));
+                    codeFragment.Add(string.Format(
+                        FORMAT_STR,
+                        Program[index].SourceName,
+                        Program[index].SourceLine,
+                        Program[index].SourceColumn,
+                        index,
+                        Program[index].Label,
+                        Program[index],
+                        (index == InstructionPointer ? "<<--INSTRUCTION POINTER--" : "")));
```

Comment:
After adding yet another parameter I decided this was getting absurdly long for one line.